src/renderers/shared/devtools/__tests__/ReactComponentTreeDevtool-test.js

```diff
@@ -1716,103 +1716,137 @@ describe('ReactComponentTreeDevtool', () => {
     expect(ReactComponentTreeTestUtils.getRegisteredDisplayNames()).toEqual([]);
   });
 
-  it('creates current stack addenda', () => {
-    function getAddendum(element) {
-      var addendum = ReactComponentTreeDevtool.getCurrentStackAddendum(element);
-      return addendum.replace(/\(at .+?:\d+\)/g, '(at **)');
-    }
+  describe('stack addenda', () => {
+    it('gets created', () => {
+      function getAddendum(element) {
+        var addendum = ReactComponentTreeDevtool.getCurrentStackAddendum(element);
+        return addendum.replace(/\(at .+?:\d+\)/g, '(at **)');
+      }
 
-    var Anon = React.createClass({displayName: null, render: () => null});
```

Comment:
These changes are just indentation. I moved an existing test into a common `describe` and added two more tests based on the issue reports.